### PR TITLE
[10.0][MIG] business_product_location

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
Migration from v8

File: business_product_location/models/business_product_line.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        """ Change UoM if product_id changes
+        """
+        if self.product_id:
+            self.product_uom_id = self.product_id.uom_id
+
+    @api.onchange('product_uom_id')
+    def _onchange_product_uom_id(self):
+        """ Check the selected UoM with the product UoM
+        """
+        res = {}
```
Comment (andreparames) [inline business_product_location/models/business_product_line.py:51]: `res` should be returned (this is a problem coming from the original module)

Comment (rousseldenis) [reply business_product_location/models/business_product_line.py:51]: @andreparames Of course ! I will add a test too

**Review by andreparames — APPROVED** [review]

**Comment by rousseldenis** [comment]
@andreparames

**Comment by rousseldenis** [comment]
@chrisdec A review on this ?

**Review by chrisdec — APPROVED** [review]
functional tests

**Pull request merged**